packages/ember-metal/lib/computed.js

```diff
@@ -548,12 +551,19 @@ export default function computed(...args) {
 function cacheFor(obj, key) {
   let meta = peekMeta(obj);
   let cache = meta && meta.source === obj && meta.readableCache();
-  let ret = cache && cache[key];
 
-  if (ret === UNDEFINED) {
-    return undefined;
+  if (cache !== undefined) {
+    let prop = obj[key];
+    if (prop instanceof AliasedProperty) {
```

Comment:
could check for existence of `prop.altKey` instead of `instanceof` for performance reasons